src/scripts/createClaudeDesktopExtensionManifest.ts

```diff
@@ -0,0 +1,218 @@
+    required: false,
+    sensitive: false,
+  },
+  DEFAULT_LOG_LEVEL: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'Default Log Level',
+    description: 'The default logging level of the server.',
+    required: false,
+    sensitive: false,
+  },
+  DATASOURCE_CREDENTIALS: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'Datasource Credentials',
+    description:
+      'A JSON string that includes usernames and passwords for any datasources that require them.',
+    required: false,
+    sensitive: true,
+  },
+  INCLUDE_TOOLS: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'Included Tools',
+    description:
+      'A comma-separated list of tool names to include in the server. Only these tools will be available.',
+    required: false,
+    sensitive: false,
+  },
+  EXCLUDE_TOOLS: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'Excluded Tools',
+    description:
+      'A comma-separated list of tool names to exclude from the server. All other tools will be available.',
+    required: false,
+    sensitive: false,
+  },
+  MAX_RESULT_LIMIT: {
+    includeInUserConfig: false,
+    type: 'number',
+    title: 'Max Result Limit',
+    description:
+      'If a tool has a "limit" parameter and returns an array of items, the maximum length of that array.',
+    required: false,
+    sensitive: false,
+  },
+  DISABLE_QUERY_DATASOURCE_FILTER_VALIDATION: {
+    includeInUserConfig: false,
+    type: 'boolean',
+    title: 'Disable Query Datasource Filter Validation',
+    description: 'Disable validation of SET and MATCH filter values in query-datasource tool.',
+    required: false,
+    sensitive: false,
+  },
+  SSL_KEY: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'SSL Key Path',
+    description: 'The path to the SSL key file to use for the HTTP server.',
+    required: false,
+    sensitive: false,
+  },
+  SSL_CERT: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'SSL Certificate Path',
+    description: 'The path to the SSL certificate file to use for the HTTP server.',
+    required: false,
+    sensitive: false,
+  },
+  HTTP_PORT_ENV_VAR_NAME: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'HTTP Port Environment Variable Name',
+    description: 'The environment variable name to use for the HTTP server port.',
+    required: false,
+    sensitive: false,
+  },
+  CORS_ORIGIN_CONFIG: {
+    includeInUserConfig: false,
+    type: 'string',
+    title: 'CORS Origin Config',
+    description: 'The origin or origins to allow CORS requests from.',
+    required: false,
+    sensitive: false,
+  },
+  DISABLE_LOG_MASKING: {
+    includeInUserConfig: false,
+    type: 'boolean',
+    title: 'Disable Log Masking',
+    description: 'Disable masking of credentials in logs. For debug purposes only.',
+    required: false,
+    sensitive: false,
+  },
+} satisfies EnvVars;
```

Comment:
The point of this is any time we add a new environment variable in process-env.d.ts, the build will fail if we forget to update this object.

```
Property 'MY_NEW_ENV_VAR' is missing in type ...
```